
<launch>
  
  <node pkg="tf" type="static_transform_publisher" name="world_to_cam" 
    args="0 0 0.5 -1.57 0 -1.57 world camera 10" />
  
  
  
  
  
  <node ns="camera_rot" pkg="libuvc_camera" type="camera_node" name="camera_node"
    output="screen">
    <remap from="camera_info" to="/camera/camera_info"/>
    <param name="width" type="int" value="640" />
    <param name="height" type="int" value="480" />
    <param name="fps" type="int" value="30" />
    <param name="frame" type="string" value="camera" />
    <param name="device" type="string" value="/dev/video1" />
    <param name="camera_info_url" type="string"
      value="file:///home/tman/.ros/camera_info/camera.yaml" />
  </node>
  
  
  
  <node pkg="image_flip" type="image_flip" name="image_flip" args="b" output="screen">
   <remap from="flipped" to="/camera/image_raw"/>
   <remap from="image" to="/camera_rot/image_raw"/>
  </node>

</launch>
